src/Xrefcheck/Config.hs

```diff
@@ -72,3 +74,26 @@ deriveFromJSON defaultOptions ''VerifyConfig
 instance KnownRatName unit => FromJSON (Time unit) where
     parseJSON = withText "time" $
         maybe (fail "Unknown time") pure . unitsP . toString
+
+instance FromJSON Regex where
+    parseJSON = withText "regex" $ \val -> do
+        let errOrRegex =
+                compile defaultCompOption defaultExecOption val
+        either (error . show) return errOrRegex
+
+-- Default boolean values according to
+-- https://hackage.haskell.org/package/regex-tdfa-1.3.1.0/docs/Text-Regex-TDFA.html#t:CompOption
+defaultCompOption :: CompOption
+defaultCompOption =
+    CompOption
+    { caseSensitive = True
+    , multiline = True
+    , rightAssoc = True
+    , newSyntax = True
+    , lastStarGreedy = False
+    }
+
+-- Default boolean value according to
+-- https://hackage.haskell.org/package/regex-tdfa-1.3.1.0/docs/Text-Regex-TDFA.html#t:ExecOption
+defaultExecOption :: ExecOption
+defaultExecOption = ExecOption {captureGroups = False}
```

Comment:
Do I understand correctly that `False` is not the default value provided by regex-tdfa as mentioned in the comment above, but we pass it to speed up things (which seems reasonable)?

Reply:
Oh, you are right. I've just "copied" this comment from the previous default value.